### PR TITLE
Fix #98 and remove package-lock.json

### LINKED ISSUE #98: Upgrade node-fetch to >= 2.3.0 to fix semantic-release users that use styled-components
## Environment

We are experiencing issue on travis while using semantic-release.

This happen because of a package having being fixed `node-fetch@2.3.0` for semantic-release to work and styled-components having an outdated node-fetch through [isomorphic-fetch](https://github.com/matthew-andrews/isomorphic-fetch/issues/new).

Users that use both semantic-release and styled-components will have issues. 

- https://github.com/semantic-release/github/issues/129
- https://github.com/semantic-release/semantic-release/issues/995

## Related issues:

- https://github.com/styled-components/styled-components/issues/2247#issuecomment-442114941
- https://github.com/styled-components/css-to-react-native/issues/98
- https://github.com/facebook/fbjs/issues/325
- https://github.com/matthew-andrews/isomorphic-fetch/issues/179

## Reproduction

https://travis-ci.org/yeutech-lab/rollup-umd-documentation/jobs/459847412

```bash
$ npm ls node-fetch
@yeutech-lab/rollup-umd-documentation@2.4.2 /home/dka/workspace/github.com/yeutech-lab/rollup-umd-documentation
├─┬ @semantic-release/github@5.2.5
│ └─┬ @octokit/rest@16.0.1
│   └── node-fetch@2.3.0 
└─┬ styled-components@4.1.1
  └─┬ css-to-react-native@2.2.2
    └─┬ fbjs@0.8.17
      └─┬ isomorphic-fetch@2.2.1
        └── node-fetch@1.7.3 
```

## Steps to reproduce

- semantic-release: v15.12.1
- styled-components: v4.1.1

## Expected Behavior

To have `node-fetch@2.3.0`

## Actual Behavior

Styled-components bring `node-fetch@1.7.3` and breaks semantic-release.

## Solution

- **short term**: `npm install node-fetch@2.3.0 --save-dev`
- **long term**: Upgrade chain of dependencies, require help of https://github.com/matthew-andrews/isomorphic-fetch/, https://github.com/facebook/fbjs, https://github.com/styled-components/css-to-react-native/ and https://github.com/styled-components/styled-components

### PR DESCRIPTION
**Pull request by kopax**
This fix:

- #98 
- test: `npm test` was failing due to `jest.testURL` not specified in `package.json`
- lockfile: lock file should not be used for modules, only for application. This can lead to potential errors. It was remove in profit of `.npmrc` telling this does not need a `package-lock.json`

### CODE REVIEW

**Comment by jacobp100** [comment]
Could you update the yarn lock file? 😀

**Comment by kopax** [comment]
```bash
$ yarn
bash: yarn: command not found
$ npx yarn
npx: installed 1 in 5.468s
yarn install v1.12.3
# ... zzz
To github.com:kopax/css-to-react-native.git
   c62fdfb..5bb1433  master -> master
```

All good

File: package.json [hunk 1]
```diff
@@ -33,6 +33,9 @@
     "url": "https://github.com/styled-components/css-to-react-native/issues"
   },
   "homepage": "https://github.com/styled-components/css-to-react-native#readme",
+  "jest": {
+    "testURL": "http://localhost"
```
Comment (jacobp100) [inline package.json]: What's this?

Comment (kopax) [reply package.json]: Basically, it use a vhost on the user to connect on my server. Just kidding This is for jest. Try to pull your repo and you will see error such as `localStorage cannot get ....`. We got this in our projects long time ago, seems like you haven't tried it for a while. Try to pull your branch, remove `yarn.lock` and run `npm install && npm test`. You will see.

Comment (kopax) [reply package.json]: By the way, I've removed the `package-lock.json`, I haven't seen the `yarn.lock`, but it should also be removed from the repo. You don't want to use lock file when you maintain a module.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 /node_modules
 /index.js
+/package-lock.json
```
Comment (stevemao) [inline .gitignore]: Is this needed when you have `package-lock=false` in `.npmrc`?

**Pull request merged**